papis/commands/doctor.py

```diff
@@ -123,7 +123,7 @@ class Error(NamedTuple):
     #: A command to run to fix the error that can be suggested to the user.
     suggestion_cmd: str
     #: A callable that can autofix the error.
-    fix_action: FixFn
+    fix_action: Optional[FixFn]
```

Comment:
This also signals whether there is a `fix_action` avilable, i.e., `fix_action is None` implies there is no `fix_action`. With `lambda: None` there is no such signal. This allows you to e.g., print a warning if no fix action is available. I use that in auto-doctor to decide whether to launch `edit` automatically. If every error returned by the checkers is fixed with `fix_action` no need to `edit` - otherwise launch the editor.

Reply:
Ah, right, missed that part of the logic! Thanks for explaining!